docs/guide/acquiring_data/fido.rst

```diff
@@ -180,131 +167,346 @@ client for results, in this case VSO.::
+    2011-01-01 00:05:40.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+    2011-01-01 00:06:25.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+    2011-01-01 00:07:10.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+    2011-01-01 00:07:55.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+    2011-01-01 00:08:40.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+    2011-01-01 00:09:25.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
+
+To match the ``"Wavelength"`` column we need to account for the fact that VSO results return a wavelength range of ``[min, max]`` so we match the min::
+
+   >>> aia_94 = aia[aia["Wavelength"][:, 0] == 94 * u.AA]  # doctest: +REMOTE_DATA
+   >>> aia_94.show("Start Time", "Instrument", "Wavelength", "Physobs")  # doctest: +REMOTE_DATA
+   <sunpy.net.vso.table_response.VSOQueryResponseTable object at ...>
+          Start Time       Instrument Wavelength [2]  Physobs
+                                         Angstrom
+   ----------------------- ---------- -------------- ---------
+   2011-01-01 00:00:02.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:00:14.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:00:26.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:00:38.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:00:50.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:01:02.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:01:14.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:01:26.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:01:38.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:01:50.000        AIA   94.0 .. 94.0 intensity
+                       ...        ...            ...       ...
+   2011-01-01 00:08:02.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:08:14.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:08:26.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:08:38.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:08:50.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:09:02.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:09:14.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:09:26.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:09:38.000        AIA   94.0 .. 94.0 intensity
+   2011-01-01 00:09:50.000        AIA   94.0 .. 94.0 intensity
+   Length = 50 rows
+
+These can then be passed to `Fido.fetch <sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch>`::
+
+    >>> Fido.fetch(hmi_los, aia_94)  # doctest: +SKIP
+
+.. warning::
+
+   While you can reduce the number of columns and rows in the results, the
+   ``fetch()`` method may need certain columns to be present to successfully
+   download the files. It is therefore highly recommended that if you are
+   planning on downloading data you do not slice out columns, but instead use
+   ``.show()`` to only display the ones you are interested in.
```

Comment:
this is a bit unclear what you mean - what is the recommended thing to do to pass to `.fetch`?

Reply:
What I am trying to say here could be summarised as "don't slice out columns from the response, fetch might need them, if you want to view the table with less columns use `.show()`"